src/olympia/devhub/templates/devhub/new-landing/index.html

```diff
@@ -1 +1,64 @@
-New DevHub Landing
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% if not settings.ENGAGE_ROBOTS %}
+      <meta name="robots" content="noindex">
+    {% endif %}
+    {% block extrameta %}{% endblock %}
+
+    <title>{% block title %}{{ dev_page_title() }}{% endblock %}</title>
+
+    <link rel="shortcut icon" type="image/x-icon"
+          href="{{ STATIC_URL }}img/favicon.ico">
+
+    {% block rss_feed %}{% endblock %}
+
+    {# L10n: {0} is an application, like Firefox. #}
+    <link title="{{ _('{0} Add-ons')|f(APP.pretty) }}"
+          rel="search" type="application/opensearchdescription+xml"
+          href="{{ url('amo.opensearch') }}" />
+
+    {% block site_css %}
+      {{ css('devhub/css') }}
+    {% endblock %}
+
+    {% block extrahead %}{% endblock %}
+
+    {% if request.user.is_authenticated() %}
+      <meta name="csrf" content="{{ csrf_token }}">
+    {% endif %}
+
+    {{ js('preload') }}
+  </head>
+  <body class="html-{{ DIR }} {{ request.APP.short }} {% block bodyclass %}{% endblock %}"
+        data-raven-dsn="{{ settings.RAVEN_DSN }}"
+        data-raven-urls="{{ settings.RAVEN_ALLOW_LIST }}"
+        {% block bodyattrs %}{% endblock %}>
+
+    <nav class="devhub-navigation">
+      <ul>
+        <li><a href="#"><img src="{{ STATIC_URL }}img/developers/addons-logo.svg"></a></li>
```

Comment:
Imho a logo shouldn't be part of the list.

If this svg remains an image wrapped with a link it must have a non-empty alt attribute.

In the mobile pages we're using "Firefox Addons", it will (eventually be) an h1 without a link on the homepage, and everywhere else a link to the hompage and the h1 then becomes the heading of the document.

In your case /developers/ is a homepage or sorts, so presumably "clicking the logo" should link to that except when you're on that page.